src/App.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import './css/App.css';
+import Header from './Header';
+import Timer from './Timer';
+import Image from './Image';
+import {TimerConstants, MAX_SESSION_TIME, MIN_SESSION_TIME, MAX_BREAK_TIME, MIN_BREAK_TIME} from './Constants';
+
+class App extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            breakLength:TimerConstants.breakLength,
+            sessionLength:TimerConstants.sessionLength
+        }
+        this.increaseSessionLength = this.increaseSessionLength.bind(this);
```

Comment:
you no longer need to bind your functions with this as you are using arrow functions.
[https://reactjs.org/docs/faq-functions.html#how-do-i-bind-a-function-to-a-component-instance](https://reactjs.org/docs/faq-functions.html#how-do-i-bind-a-function-to-a-component-instance)

Reply:
this question still exists